app/models.py

```diff
@@ -288,6 +288,19 @@ class Config:
         }
 
 
+class QuizCreationResponse(BaseModel):
```

Comment:
Also can you update this to `CreateQuizResponse`?
That way, you can also update the model being used in get quiz as `GetQuizResponse`.
So it will be easier to extend if we create an update method in the future.
create - CreateQuizResponse
get - GetQuizResponse
... in the future ...
update - UpdateQuizResponse etc

Reply:
yes, updated